packages/steiger-plugin-fsd/src/no-cross-imports/index.spec.ts

```diff
@@ -0,0 +1,191 @@
+import { expect, it, vi } from 'vitest'
+
+import { joinFromRoot, parseIntoFolder as parseIntoFsdRoot } from '@steiger/toolkit/test'
+import forbiddenImports from './index.js'
```

Comment:
issue: this rule name is probably left over from copy-paste

suggestion:
```suggestion
import noCrossImports from './index.js'
```